server/src/main.rs

```diff
@@ -67,7 +98,7 @@ fn main() {
         // The errors are consciously ignored as the program is terminating.
         let binary_path = std::env::current_exe().unwrap();
         let binary_name = binary_path.file_name().unwrap();
-        println!("usage: {} data_folder", binary_name.to_str().unwrap());
+        error!(" Usage: {} data_folder ", binary_name.to_str().unwrap());
```

Comment:
Why is there an added space at the end of this log message?

Reply:
Fixed